test/parallel/test-http2-server-timeout.js

```diff
@@ -6,14 +6,9 @@ if (!common.hasCrypto)
 const http2 = require('http2');
 
 const server = http2.createServer();
-server.setTimeout(common.platformTimeout(50));
```

Comment:
Shouldn't we keep this as a separate test in this file since `server.setTimeout()` should still work?

Reply:
At first I've moved it into a separate test, but I agree that it doesn't make a lot of sense in this case. Introduced additional assert in this very test.

Reply:
I think what was meant was a separate test file. Calling `server.setTimeout()` then immediately resetting it with `server.timeout` does not really ensure that `server.setTimeout()` does the right thing.

Reply:
Original version of this PR included a separate test, but I have removed it and merged everything into a this test after receiving this comment.

@mscdex could you check current code and tell me if that's what you wanted?

Reply:
@mscdex could you check current code and tell me if that's what you wanted? I'm thinking of reverting this change, as it seems more logical to me to keep tests isolated.

Reply:
I have updated the test, so now it verifies both `timeout` property and `setTimeout` method in isolation from each other. Hopefully, it resolves this comment.